editor/src/messages/portfolio/document/overlays/utility_types_vello.rs

```diff
@@ -1046,6 +1048,8 @@ impl OverlayContextInternal {
 		};
 
 		// Load Source Sans Pro font data
+		// TODO: Grab this from the node_modules folder (either with `include_bytes!` or ideally at runtime) instead of checking the font file into the repo.
+		// TODO: And maybe use the WOFF2 version (if it's supported) for its smaller, compressed file size.
```

Comment:
Woff didn't work I tried that before using the ttf version

Reply:
We should probably aim to load this dynamically then instead of inlining it into the binary.